### ISSUE TITLE
Enums 2 Exercise issue

### ISSUE THREAD

**Issue by gianalarcon** [shramee/starklings-cairo1#177]
Trying to run the exercise enums2.cairo with the following solution...

```rust
enum Message { // TODO: define the different variants used below
    Quit,
    Echo: felt252,
    Move: (felt252, felt252),
    ChangeColor: (felt252, felt252, felt252),
}
```

...fails with the following error message...
```console
#47: Expected variable data for statement not found.
```

According to my own research it is a bug introduced since commit 16b4790aa0f68fedfbaa921e76032650ced486c1

**Comment by swarajbachu** [comment]
I am having the same issue, and It did after the commit mentioned, coz I wasnt having it before, when I tried to sync it back, I am facing the issue @shramee

**Comment by swarajbachu** [comment]
and I guess most probably it is from 
```use debug::PrintTrait;``` this library

**Comment by gianalarcon** [comment]
Yes, In my case I commented all the helper functions related to print

**Comment by swarajbachu** [comment]
I did the same

**Comment by shramee** [comment]
Hi @gianalarcon,
Just fixed this in #181. Are you able to help review this?

**Comment by gianalarcon** [comment]
Yes. It is working with #181
